Patches/Simple Ogre Race/CE_Ogre_Race.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<!-- It's all Ogre now -->
+	<Operation Class="PatchOperationFindMod">
+					<mods>
+						<li>Simple Ogre Race</li>
+					</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="FP_OgreRace"]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Humanoid</bodyShape>
+					</li>
+				</value>
+			</li>
+			
+			<!-- Melee Tool & Basestats Defs -->
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="FP_OgreRace"]/statBases/Mass</xpath>
+					<value>
+      						<Mass>80</Mass>
+						<MeleeDodgeChance>1</MeleeDodgeChance>
+						<MeleeCritChance>1.5</MeleeCritChance>
+						<MeleeParryChance>1</MeleeParryChance>
+          						<Suppressability>0.75</Suppressability>
+						<CarryWeight>55</CarryWeight>
+			                			<CarryBulk>30</CarryBulk>
+					</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="FP_OgreRace"]/statBases/MoveSpeed</xpath>
+				<value>
+					<MoveSpeed>4.0</MoveSpeed>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="FP_OgreRace"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>0.2</ArmorRating_Sharp>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="FP_OgreRace"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>2</ArmorRating_Blunt>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="FP_OgreRace"]/statBases/ArmorRating_Heat</xpath>
+				<value>
+					<ArmorRating_Heat>0.1</ArmorRating_Heat>
+				</value>
+			</li>
+
+		        <li Class="PatchOperationReplace">
+				<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="FP_OgreRace"]/race/baseHealthScale</xpath>
+					<value>
+						<baseHealthScale>2</baseHealthScale>
+					</value>
+			</li>
+
+		        <li Class="PatchOperationReplace">
+				<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="FP_OgreRace"]/race/baseBodySize</xpath>
+					<value>
+						<baseBodySize>1.5</baseBodySize>
+					</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="FP_OgreRace"]/tools</xpath> 
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>left fist</label>
+							<capacities>
+								<li>Blunt</li>
+							</capacities>
+							<power>14</power>

```

Comment:
Be aware that since the ogres have Melee skill, their damage output increases above 14 as their skill gets better. So, a level 13 ogre's punches do 21 damage per hit.

Given that it will scale with their level, it may be better to decrease it to around 2 - 4.

Reply:
Unarmed dmg increase seem to work the last time I tested it on nonbaseline humans, I'll test it again.

Reply:
Come to think of it, what is the criteria for the unarmed damage increase? Does it only work for fist attacks? what about head, bite or bionic limb attacks?